src/emq_web_hook.erl

```diff
@@ -225,15 +225,40 @@ on_message_acked(ClientId, Username, Message = #mqtt_message{topic = Topic}, {Fi
 %% Internal functions
 %%--------------------------------------------------------------------
 
+-spec get_timestamp_ms() -> integer().
+get_timestamp_ms() ->
+    {Mega, Sec, Micro} = os:timestamp(),
+    (Mega*1000000 + Sec)*1000 + round(Micro/1000).
+
 send_http_request(Params) ->
     Params1 = iolist_to_binary(mochijson2:encode(Params)),
     Url = application:get_env(?APP, url, "http://127.0.0.1"),
     ?LOG(debug, "Url:~p, params:~s", [Url, Params1]),
-    case httpc:request(post, {Url, [], "application/json", Params1}, [{timeout, 5000}], []) of
-        {ok, _} -> 
+    case perform_request(post, {Url, [], "application/json", Params1}, [{timeout, 5000}], [], 1, 4) of
```

Comment:
Is it easy to parameterize MaxAttempt? Also the backoff of 10 seconds that you put down below. It's ok to use 4 and 10 as default values.

Reply:
I think I can put this values in env vars, good point!

Reply:
We decided to not implement the retries right now!